topics/admin/tutorials/reference-genomes/tutorial.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: tutorial_hands_on
+
+title: "Reference Data with Data Managers"
+zenodo_link: ""
+questions:
+objectives:
+  - Have an understanding of the way in which Galaxy stores and uses reference data
+  - Be able to download and use data managers to add a reference genome and its pre-calculated indices into the Galaxy reference data system
+  - Use an Ansible playbook for all of the above
+time_estimation: "1h"
+key_points:
+  - Understand how Galaxy stores and uses its reference data
+  - Understand how to manually add a reference genome and tool indices if required
+  - Understand and how to use data managers to make all of this much much easier
+contributors:
+  - Slugger70
+  - afgane
+  - natefoo
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Reply:
Thanks - I am not sure about the exact contributions. From what I can tell about [the blame](https://github.com/galaxyproject/admin-training/blame/2018-oslo/sessions/05-reference-genomes/ex1-reference-genomes.md) the original was written by Simon, with later additions and updates from Enis.